### PR TITLE
Add Angular UI

### PR DESCRIPTION
**Pull request by pst**

### CODE REVIEW

**Comment by democratic-collaboration** [comment]
[democratic collaboration](https://github.com/TooAngel/democratic-collaboration)
Approved reviews will speed up the merge, request changes will slow it down.

Please review the PR to help.

**Comment by democratic-collaboration** [comment]
Adding WIP label, the title is prefixed with [WIP]

**Comment by democratic-collaboration** [comment]
Adding WIP label, the title is prefixed with [WIP]

**Comment by democratic-collaboration** [comment]
Removing WIP label, the title is not prefixed with [WIP]

File: src/server.py [hunk 1]
```diff
@@ -20,7 +20,10 @@
     template_folder='../templates',
     static_folder='../static'
 )
+
 api = restful.Api(app)
+
+app.config['MONGO_URI'] = os.getenv('MONGO_URI')
```
Comment (TooAngel) [inline src/server.py]: A default localhost value would be nice.

**Comment by democratic-collaboration** [comment]
A new review, yeah.

    Votes: 71/72
    Coefficient: 0.986111111111
    Merging in 0 days 9 hours
    Age 0 days 0 hours

**Comment by democratic-collaboration** [comment]
A new review, yeah.

    Votes: 71/72
    Coefficient: 0.986111111111
    Merging in 0 days 3 hours
    Age 0 days 0 hours

**Comment by democratic-collaboration** [comment]
A new review, yeah.

    Votes: 71/72
    Coefficient: 0.986111111111
    Merging in 0 days 3 hours
    Age 0 days 0 hours

**Pull request merged**